packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/Servers.tsx

```diff
@@ -31,9 +32,21 @@ export default function Container() {
   return <ServerList {...state} />;
 }
 
+function getEmptyTableText(status: AttemptStatus) {
+  switch (status) {
+    case 'error':
+      return 'Failed to fetch servers.';
+    case '':
+      return 'Searching…';
+    case 'processing':
+      return 'Searching…';
+    case 'success':
+      return 'No servers found.';
+  }
+}
```

Comment:
To be honest, I still don't understand why this function needs to be copied three times instead of once and accepting the parts that change through args. 😶‍🌫️ From what I can tell, it's only the noun that's different.